README.md

```diff
@@ -37,7 +36,7 @@ and feature (OTU) tables we have included [here](https://github.com/biota/source
 
 ## API
 For descriptions of the requirements, please see documentation in the `gibbs`
-function. Very briefly, this function wraps the main workhorse function `_gibbs_sampler`
+function. Very briefly, this function wraps the main workhorse function `gibbs_sampler`
```

Comment:
I'd drop the *Very briefly, ...*, again because this is a document for the user who doesn't know/care about private functionality.